mongorestore/restore.go

```diff
@@ -147,6 +147,7 @@ func (restore *MongoRestore) RestoreIntents() Result {
 			totalResult.combineWith(result)
 			if totalErr == nil && totalResult.Err != nil {
 				totalErr = totalResult.Err
+				return totalResult.withErr(totalErr)
```

Comment:
Without this line, the function will still exit in the return statement below, right? What's the change here?